cli/command/image/prune.go

```diff
@@ -55,7 +55,7 @@ Are you sure you want to continue?`
 Are you sure you want to continue?`
 )
 
-func runPrune(dockerCli *command.DockerCli, opts pruneOptions) (spaceReclaimed uint64, output string, err error) {
+var runPrune = func(dockerCli command.Cli, opts pruneOptions) (spaceReclaimed uint64, output string, err error) {
```

Comment:
We shouldn't do that 😅 making this `fun` a variable so we can override it for *testing* purpose doesn't feel right for me 👼  — this also means that, for some tests, we don't test/exercice/cover this function…